test/DynamoCoreWpfTests/GroupStylesTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Dynamo.Configuration;
+using Dynamo.Nodes;
+using Dynamo.Utilities;
+using Dynamo.ViewModels;
+using Dynamo.Views;
+using Dynamo.Wpf.Views;
+using DynamoCoreWpfTests.Utility;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
```

Comment:
Same here and other places, please check in this PR

Reply:
Reorganizing usings in the next files (due that a setting from the CodingStandards was missing):
- StyleItem.cs
- GroupStyleItemSelector.cs
- AnnotationViewModel.cs
- PreferencesView.xaml.cs
- GroupStylesTests.cs
See details in commit: https://github.com/DynamoDS/Dynamo/pull/12667/commits/5fc4b1e35ba1ad5b280d4b412fa73f762bed056e